metomi/rose/date_cli.py

```diff
@@ -135,7 +135,7 @@
     * ctime: `%a %b %d %H:%M:%S %Y`
     * Unix date: `%a %b %d %H:%M:%S %Z %Y`
```

Comment:
Are these not deprecated too?

(by which I mean does `isodatetime` support them)

Reply:
Supported: https://github.com/metomi/isodatetime/blob/2e7cea4ee01c136e5077105ec5a8872b7df69f5c/metomi/isodatetime/datetimeoper.py#L41-L47